apps/src/gamelab/animationLibrary.json

```diff
@@ -0,0 +1,332 @@
+{
+  "metadata": {
+    "asterisk_circle": {
+      "name": "asterisk_circle",
+      "aliases": [
+        "p5.play"
```

Comment:
I really like that we have aliasing, because some of these names might be hard to match in a search. (ex - I can see a student searching `character` or something very general)